ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/flow_tracking.py

```diff
@@ -34,7 +34,7 @@ def flow_tracking(self) -> dict | None:
             "hardware": {
                 "trackers": [
                     {
-                        "name": self.shared_utils.wan_flow_tracker_name,
+                        "name": self.shared_utils.flow_tracker_name,
```

Comment:
I think you need to change this further to match the flow_tracking_Settings new structure, maybe a for loop on the trckers and only if there is no settings render the default one.

Also it may not be enough now to check for `is_cv_pathfinder_router`

Reply:
Yes that is done, now.